packages/Responsive/Responsive.md

```diff
@@ -1,18 +1,18 @@
 The `Responsive` component is a thin wrapper over the [**react-media**](https://github.com/ReactTraining/react-media) community
 component, which "listens for matches to a CSS media query and renders its child component based on whether the query matches or not."
 
-This package also contains Sass media query methods (documented below). In most cases **we recommend using the Sass methods over the Responsive React component** since they do not adversely affect server-side rendering. While not recommended for general use, the Responsive component is useful if you want to do something programmatically. For example, changing a prop or swapping a component on resize.
+The package also contains a `media` utility (documented below). In most cases **we recommend using the media utility over the Responsive React component** since they do not adversely affect server-side rendering. While not recommended for general use, the Responsive component is useful if you want to do something programmatically. For example, changing a prop or swapping a component on resize.
 
 ### Breakpoints
 
-The included Sass-based media queries and the `Responsive` component both use the same breakpoint keys. The following chart details these breakpoint keys, and what values they represent.
+The included `media` utility and the `Responsive` component both use the same breakpoint keys. The following chart details these breakpoint keys, and what values they represent.
 
-| Breakpoint key | `min-width` / `$from` Value | `max-width` / `$until` Value |
```

Comment:
Please copy this table to the Sass documentation below, that way consumers can still reference the `$from` and `$until` values.